doccano_api_client/beta/controllers/example.py

```diff
@@ -88,27 +94,16 @@ def get(self, example_id: int) -> ExampleController:
             client_session=self.client_session,
         )
 
-    def all(self) -> Iterable[ExampleController]:
+    def all(self, confirmed=None) -> Iterable[ExampleController]:
         """Return a sequence of Examples for a given controller, which maps to a Project"""
-        example_obj_fields = set(example_field.name for example_field in fields(Example))
-        # Examples are paginated in Doccano's API, so handle pagination with yields
-        page_index = 0
-        example_dicts = self._get_examples_response_from_api(page_index=page_index)["results"]
-
-        # When there are no more examples to paginate through, "results" recieved will be empty
-        # NOTE: The Doccano pagination API responses also have "next" field that can be used to
-        #       query the next page url. The problem is, this is returning urls with "http"
-        #       as the protocol even when the host is https, so we use this method for
-        #       pagination instead
-        #       This still works deterministically because on the API side, the example list
-        #       view/api endpoint is explicitly ordered. For non-randomized example order, it's
-        #       it's ordered by ID. For randomized order, it's ordered randomly but with a fixed
-        #       seed (the requesting user's id).
-        while len(example_dicts) != 0:
-            # Given a single page, iterate through the dicts on that page
-            for example_dict in example_dicts:
-                # TODO (louis): make a Example.from_json() method to handle this. Also, do this
-                #               for all other models
+        response = self.client_session.get(f"{self.examples_url}?confirmed={confirmed}")
+
+        while True:
```

Comment:
You may disagree, and let me know if you do, but I'm not a fan of using `while True` here if avoidable. Based on the use of `break` down below, you can probably rewrite this without needing `while True` fairly easily.